src/views/LogoutLedger.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+    <div class="container">
+        <SmallPage>
+            <PageHeader>Mind the loss of settings</PageHeader>
+            <PageBody>
+                <div class="warning nq-text nq-red">
+                    Logging out will delete custom settings and labels. Contacts are not affected.
```

Comment:
Clarify that it deletes stuff _for this account only_, and I would leave out the contacts part, because contacts are not yet stored in Accounts.
```suggestion
                    Logging out will delete custom settings and labels for this account.
```

Reply:
The contacts part is needed as it's not clear to the user wheere contacts are stored or related to.
Tammo and Julian were actually surprised that the contacts persist.
It doesn't make a lot of sense for third party apps without contacts though.

Reply:
I changed it to only display the note about contacts when coming from the safe